config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/dishes/:id", to: "dishes#show"
+  post "/dishes/:id/ingredients", to: "ingredient_dishes#create"
 end
```

Comment:
Great restful routing and sending the request to the proper controller for joining an ingredient to a dish.